tests/functests/instancetype_test.go

```diff
@@ -272,6 +272,9 @@ var _ = Describe("Common instance types func tests", func() {
 			Entry("[test_id:????] Windows 11", windows11ContainerDisk, "u1.2xmedium",
 				map[string]string{"amd64": "windows.11.virtio"},
 				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnWindows}),
+			Entry("[test_id:????] Windows Server 2003", windows2k3ContainerDisk, "u1.nano",
+				map[string]string{"amd64": "windows.2k3"},
+				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnWindows}),
```

Comment:
We could think about dropping the requirement for SSH. IMO the guest agent suffices.

Reply:
Done.

Reply:
@jcanocan Do you want also to drop it in Windows XP, in another PR later?

Reply:
Yes, I will do it in a follow-up PR. It is quite unrelated with this one.

Reply:
Here we go: https://github.com/kubevirt/common-instancetypes/pull/384